File: src/main/java/com/application/cv_application/requests/EmployeeRequest.java
package com.application.cv_application.requests;

import com.application.cv_application.enums.Position;

import java.time.LocalDate;

public record EmployeeRequest(
         String email,
         String firstName,
         String lastName,
         LocalDate dateOfBirth,
         LocalDate startDate,
         String title,
         Position position,
         Integer salary,
         Integer jopId,
         Integer clientId
) {
}

File: src/app/service/employee-request.ts
import { Position } from './position-enum';
export interface EmployeeRequest {
  email: string;
  firstName: string;
  lastName: string;
  dateOfBirth: string;
  startDate: string;
  title: string;
  position: Position;
  salary: number;
  jopId?: number | null;
  clientId?: number | null;
  isActive: boolean;
}
